hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp

```diff
@@ -223,8 +224,8 @@
             </tr>
             <% for (Pair<RegionInfo, RegionInfo> p : report.getHoles()) { %>
             <tr>
-              <td><%= p.getFirst() %></td>
-              <td><%= p.getSecond() %></td>
+              <td><span title="<%= p.getFirst() %>"><%= p.getFirst().getEncodedName() %></span></td>
```

Comment:
tooltip 🕺 I also started using tooltip in #1485

Reply:
I took a look but didn't see it. Mind pointing me to it so I do same as you?

Reply:
Oh, I mean this `"title="` in a `<span/>` serves as simple tooltip. You are doing this already here and I'm glad we use this as additional information. I also though of doing this similarly.

Reply:
Thanks for clarification (thought you were doing some bootstrap fancy tooltip thingy).